.env.template.development

```diff
@@ -0,0 +1,29 @@
+## GENERAL ##
+# When running through Docker, always use: NODE_ENV=production
+NODE_ENV=development
+
+# The fully qualified domain name for the host where you are running Praise
+# Use HOST=localhost for development
+HOST=localhost
+
+# When running in development, API is accessed on this port, default: localhost:8088
+# In production, this port is used internally only since api is accessed
+# on $HOST/api 
+API_PORT=8088
+
+# Full URL to the host (and optionally port) where the API is running
+SERVER_URL=http://localhost:8088
+
+# Full URL to the host (and optionally port) where frontend is being served
+FRONTEND_URL=http://localhost:3000
```

Comment:
It looks like `FRONTEND_URL` is only used in the discord-bot. Should it be moved to `discord-bot/.env`?

Reply:
Yes, thats what I thought. Most likely we will need it elsewhere.